tests/nextflow.config

```
@@ -1,4 +1,17 @@
-// Limit and standardize resources for github actions and reproducibility
+/*
+========================================================================================
+    Nextflow config file for running nf-test tests
+========================================================================================
+*/
+
+
+// Or any resources requirements
+params.modules_testdata_base_path = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/'
+params.pipelines_testdata_base_path = 'https://raw.githubusercontent.com/nf-core/test-datasets/refs/heads/differentialabundance'
+
+aws.client.anonymous = true // fixes S3 access issues on self-hosted runners
```

```suggestion
// fixes S3 access issues on self-hosted runners
aws.client.anonymous = true
```

at some point the language server might complain with inline comments